src/test/channelFilterRouteSpec.js

```diff
@@ -0,0 +1,64 @@
+var routes = [
+  'content/search',
+  'course/search',
+  'framework/category/search',
+  'framework/term/search',
+  'search'
+]
+
+var requestMiddleware = require('../middlewares/request.middleware')
+var httpMocks = require('node-mocks-http')
+
+var baseUrl = 'http://localhost:5000/v1/'
+var async = require('async')
+var _ = require('underscore')
+
+const whiteListedChannelList = process.env.sunbird_content_service_whitelisted_channels
```

Comment:
How is process.env set? Please add a TODO to fix, if this test case running correctly depends on environment.